deid/dicom/pixels/detect.py

```diff
@@ -124,7 +124,9 @@ def _has_burned_pixels_single(dicom_file, force, deid):
     # Load criteria (actions) for flagging
     filters = deid.get_filters()
     if not filters:
-        bot.exit("Deid provided does not have %filter, exiting.")
+        bot.warning("Deid provided does not have %filter.")
+        results = {"flagged": False, "results": None}
```

Comment:
What we would want to do here is move the lines under this block to be above it, e.g., these two:

https://github.com/pydicom/deid/blob/5bae2a7725dd845e24dec17e063430eec11c0470/deid/dicom/pixels/detect.py#L132

And then instead of returning None for results (will trigger an error if the user is expecting a list to loop through) and hard coding False, just do:

```python
return {"flagged": global_flagged, "results": results}
```

Reply:
Perfecto!